src/Component/ExpressionInput/NumberExpressionInput/NumberExpressionInput.tsx

```diff
@@ -92,6 +92,9 @@ export const NumberExpressionInput: React.FC<NumberExpressionInputProps> = ({
       {slider ? (
         <div className={'slider-wrapper'}>
           <Slider
+            min={0}
+            max={1}
+            step={0.01}
             value={inputValue}
             range={false}
             onChange={(val) => {
```

Comment:
It would be useful to have an `sliderProps` porperty on the NumberExpressionInput analog to the `inputProps` so you can pass props to the slider.